tripl/tripl.py

```diff
@@ -667,12 +674,13 @@ def pull(self, pull_expr, entity,
             if '*' in attr_patterns:
                 for a, vs in _entity.keys.items():
                     if a not in pull_data:
-                        pull_data[a] = vs # cardinality schema?
+                        pull_data[a] = vs  # cardinality schema?
             # Deal with the dict patterns, which correspond with relations/refs (implicit are fine; though
             # need to think about the details of how defaults and options work out)
             for dict_pattern in dict_patterns:
                 for attr, token in dict_pattern.items():
                     reverse = reverse_lookup(attr)
+                    eids = None
```

Comment:
PyCharm noticed that `eids` might be undefined below, so I explicitly set it to `None` here.